pkgs/by-name/vi/video-compare/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  SDL2,
+  SDL2_ttf,
+  ffmpeg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "video-compare";
+  version = "20250223";
+
+  src = fetchFromGitHub {
+    owner = "pixop";
+    repo = "video-compare";
+    tag = version;
+    hash = "sha256-h74TWXPcbIjfejGfHnjKZqKzpV0DIkC5llSD1jmiDDY=";
+  };
+
```

```suggestion

  postPatch = ''
    substituteInPlace makefile \
      --replace-fail 'CXX = g++' ""
  '';

```

Use c++ we provide, fix build on darwin

---

Done, thank you!